### PR TITLE
Fix conditions in examples

### PR DESCRIPTION
**Pull request by alexsnaps**
Tho they work, they rely on the lenient mode in limitador

### CODE REVIEW

**Review by guicassolato — APPROVED** [review]

File: examples/toystore/ratelimitpolicy_gateway.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ spec:
                 descriptor_key: expensive-op
                 descriptor_value: "1"
       limits:
-        - conditions: ["expensive-op == 1"]
+        - conditions: ["expensive-op == '1'"]
```
Comment (eguzki) [inline examples/toystore/ratelimitpolicy_gateway.yaml:22]: shouldn't be `expensive_op`?

File: examples/toystore/ratelimitpolicy_httproute.yaml [hunk 2]
```diff
@@ -18,7 +18,7 @@ spec:
                 descriptor_key: get-toy
                 descriptor_value: "yes"
       limits:
-        - conditions: ["get-toy == yes"]
+        - conditions: ["get-toy == 'yes'"]
```
Comment (eguzki) [inline examples/toystore/ratelimitpolicy_httproute.yaml:21]: shouldn't be `get_toy` ?

File: examples/toystore/ratelimitpolicy_httproute.yaml [hunk 3]
```diff
@@ -42,14 +42,14 @@ spec:
...
+            - "admin == 'yes'"
+            - "user-id == 'bob'"
           maxValue: 2
           seconds: 30
           variables: []
         - conditions:
-            - "admin == yes"
-            - "user-id == alice"
+            - "admin == 'yes'"
+            - "user-id == 'alice'"
```
Comment (eguzki) [inline examples/toystore/ratelimitpolicy_httproute.yaml:52]: shouldn't be `user_id` ?

File: examples/toystore/ratelimitpolicy_httproute.yaml [hunk 4]
```diff
@@ -42,14 +42,14 @@ spec:
                         key: "user-id"
       limits:
         - conditions:
-            - "admin == yes"
-            - "user-id == bob"
+            - "admin == 'yes'"
+            - "user-id == 'bob'"
```
Comment (eguzki) [inline examples/toystore/ratelimitpolicy_httproute.yaml:46]: shouldn't be `user_id` ?

**Review by eguzki — CHANGES_REQUESTED** [review]

**Comment by alexsnaps** [comment]
Good catch @eguzki !

**Review by eguzki — APPROVED** [review]

**Comment by eguzki** [comment]
by the way, do we have this limitation documented? I cannot find in https://github.com/Kuadrant/limitador/blob/main/doc/migrations/conditions.md

**Comment by alexsnaps** [comment]
>by the way, do we have this limitation documented? 
Yeah, no... it's not in there. Or say not directly. Hard to document what really has changed, might be better to just document what makes a valid identifier?

**Pull request merged**